README.md

```diff
@@ -1,33 +1,33 @@
 # Arrayzy
 
-A native PHP arrays easy manipulation library in OOP way.
+A native PHP library for manipulating arrays in an OOP way.
```

Comment:
What about to change it to:
`The wrapper for all PHP internal (built-in) array functions and easy array manipulation library on steroids in an object-oriented way.`

How is it sounds for you?